packages/@react-spectrum/buttongroup/test/ButtonGroup.test.js

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {Button} from '@react-spectrum/button';
+import {ButtonGroup} from '../';
+import {cleanup, render, within} from '@testing-library/react';
+import {Provider} from '@react-spectrum/provider';
+import React from 'react';
+import scaleMedium from '@adobe/spectrum-css-temp/vars/spectrum-medium-unique.css';
+import themeLight from '@adobe/spectrum-css-temp/vars/spectrum-light-unique.css';
+import {triggerPress} from '@react-spectrum/test-utils';
+import V2Button from '@react/react-spectrum/Button';
+import V2ButtonGroup from '@react/react-spectrum/ButtonGroup';
+
+let theme = {
+  light: themeLight,
+  medium: scaleMedium
+};
+
+let buttonGroupId = 'button-group';
+let onPressSpy1 = jest.fn();
+let onPressSpy2 = jest.fn();
+let onPressSpy3 = jest.fn();
+
+function renderComponent(Component, props = {}) {
+  if (Component === V2ButtonGroup) {
+    return render(
+      <Provider theme={theme}>
+        <V2ButtonGroup data-testid={buttonGroupId} {...props}>
```

Comment:
are any tests testing this? i think we can get rid of the V2 parity, it's pretty different in implementation no?

Reply:
I can remove the v2 stuff